src/evaluation/Evaluation.jl

```diff
@@ -5,10 +5,17 @@ using ..CounterfactualExplanations.Objectives
 using DataFrames
 using ProgressMeter
 using Statistics
+using DataFrames
+using ..Models
+using ProgressMeter
+using LinearAlgebra
+using SliceMap
 
 export Benchmark, benchmark, evaluate, default_measures
 export validity, distance, redundancy
 
+include("benchmark.jl")
+include("evaluate.jl")
 include("measures.jl")
 
```

Comment:
In your other PR (#229) you used the following order: `using`, `include`, `export` but here `export` and `include` are the other way around. Could you please change it so we'd end up using the same convention in all files?